sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -353,9 +356,10 @@ def __init__(
     self._step_name = step_name
     self._load_job_project_id = load_job_project_id
 
-  def setup(self):
-    self._bq_wrapper = bigquery_tools.BigQueryWrapper(client=self._test_client)
+  def start_bundle(self):
+    self.bq_wrapper = bigquery_tools.BigQueryWrapper(client=self._test_client)
```

Comment:
why the change between _bq_wrapper and bq_wrapper?

Reply:
Everywhere else in the file it's bq_wrapper, so I wanted to keep it consistent